src/topology/algebra/continuous_functions.lean

```diff
@@ -212,23 +212,24 @@ end subtype
 section continuous_map
 
 instance continuous_map_has_scalar {α : Type*} [topological_space α]
-  (R : Type*) [semiring R] [topological_space R]
-  (M : Type*) [topological_space M] [add_comm_monoid M]
+  {R : Type*} [semiring R] [topological_space R]
+  {M : Type*} [topological_space M] [add_comm_monoid M]
   [semimodule R M] [topological_semimodule R M] :
   has_scalar R C(α, M) :=
 ⟨λ r f, ⟨r • f, continuous_const.smul f.continuous⟩⟩
 
 instance continuous_map_semimodule {α : Type*} [topological_space α]
 {R : Type*} [semiring R] [topological_space R]
-{M : Type*} [topological_space M] [add_comm_group M] [topological_add_group M]
+{M : Type*} [topological_space M] [add_comm_monoid M] [has_continuous_add M]
 [semimodule R M] [topological_semimodule R M] :
   semimodule R C(α, M) :=
-  semimodule.of_core $
 { smul     := (•),
-  smul_add := λ c f g, by ext x; exact smul_add c (f x) (g x),
+  smul_add := λ c f g, by {ext, exact smul_add c (f x) (g x),},
```

Comment:
It's very good that you remove the semicolon and switch to braces. Normally, the style is with spaces around the braces, like
```suggestion
  smul_add := λ c f g, by { ext x, exact smul_add c (f x) (g x) },
```
Could you also do the same on the other lines?

Reply:
Haha actually I think I did this just because I needed to click in the middle of the proof to see what was going on and fill up the _ that lean was not finding. Why is it better to switch to braces? The natural thing I would do here is to go back to semicolon for coherence...

Reply:
The official mathlib style is now to use braces unless `;` is necessary to handle several goals at once, precisely to be able to follow the goals more easily. (It had not been always like that).